interface-definitions/include/accel-ppp/gateway-address-multi.xml.i

```diff
@@ -0,0 +1,17 @@
+<!-- include start from accel-ppp/gateway-address-multi.xml.i -->
+<leafNode name="gateway-address">
+  <properties>
+    <help>Gateway IP address</help>
+    <constraintErrorMessage>invalid IPv4 address</constraintErrorMessage>
+    <valueHelp>
+      <format>ipv4net</format>
+      <description>Default Gateway, mask send to the client</description>
```

Comment:
This is ungrammatical. Should be "Default gateway _to_ send to the client". Or maybe shorter: "Default gateway for clients".

Reply:
done